src/status_im/ui/screens/browser/styles.cljs

```diff
@@ -62,29 +62,38 @@
 (def web-view-error-text
   {:color colors/gray})
 
-(defnstyle toolbar-content [show-actions]
+(def toolbar-content
   {:flex-direction     :row
    :flex               1
-   :border-radius      4
-   :height             36
+   :border-radius      8
+   :max-height         36
    :background-color   colors/gray-lighter
    :padding-horizontal 12
-   :margin-right       5
+   :margin-right       8
    :align-items        :center
-   :android            {:margin-left (if show-actions 66 20)}
-   :ios                {:margin-left 20}})
+   :align-self         :center
+   :margin-top         8
```

Comment:
are you guys sure it's 8? 56 total toolbar height - 36 url bar height / 2 gives 10